crates/taplo/src/formatter/mod.rs

```diff
@@ -358,36 +359,40 @@ fn format_impl(node: SyntaxNode, options: Options, context: Context) -> String {
 struct FormattedEntry {
     syntax: SyntaxElement,
     key: String,
+    _cleaned_key: OnceCell<String>,
```

Comment:
In Rust, underscores are usually used to indicate that something is not used. I know some other languages use it to indicate something is private. But Rust already has a mechanism for that. So, let's remove the underscore. It's not possible to confuse it with the function of the same name because they have different types, if that was the concern.

Reply:
You're right. I haven't been coding for a long time, and where I do, we kinda have this convention that it can be used as either "not used" or "should not be used directly", which really is what I was going for. I realize the types make it so no confusion is possible, but it still feels like we should indicate in some way that one should not use the value directly and/or try to use its own `get_or_init` (which could potentially make for hard-to-debug issues later on). So in the end, I have now made the var entirely private :smile: . Let me know what you think.

Reply:
I think I preferred the previous version :joy: 

I don't think the additional readability cost is justified. At best, I would just add a comment on top of the field, stating it's caching the key in a format suitable for comparison and should be accessed through the helper method.